topostats/default_config.yaml

```diff
@@ -53,6 +53,13 @@ grains:
     grain_crop_padding: 2 # Padding to apply to the grain crop bounding box
     upper_norm_bound: 5.0 # Upper bound for normalisation of input data. This should be slightly higher than the maximum desired / expected height of grains.
     lower_norm_bound: -1.0 # Lower bound for normalisation of input data. This should be slightly lower than the minimum desired / expected height of the background.
+  vetting:
+    class_region_number_thresholds: null # Class region number thresholds eg 1: [2, 4], 2: [1, 1]
```

Comment:
`validation.py` indicates that these should be lists of lists of three integers, should probably have that reflected in the comment and the documentation.